js/huobi.js

```diff
@@ -239,18 +239,6 @@ module.exports = class huobi extends Exchange {
                         'etp/batch-cancel': 50, // 杠杆ETP批量撤单
                     },
                 },
-                'market': {
-                    'get': {
-                        'history/kline': 1, // 获取K线数据
-                        'detail/merged': 1, // 获取聚合行情(Ticker)
-                        'depth': 1, // 获取 Market Depth 数据
-                        'trade': 1, // 获取 Trade Detail 数据
-                        'history/trade': 1, // 批量获取最近的交易记录
-                        'detail': 1, // 获取 Market Detail 24小时成交量数据
-                        'tickers': 1,
-                        'etp': 1, // 获取杠杆ETP实时净值
-                    },
-                },
                 'public': {
```

Comment:
I can say with my responsibility, that these `market` endpoints were just of no use. all of them were for spot, and duplicated. they are defined already in `spot > public > get > `  'market/whatever'  endpoints, and used the same url in the end. So, we have to totally remove `market` tree, they are also referenced nowhere.